CHANGES/6540.feature

```diff
@@ -0,0 +1 @@
+Enable token auth sync
```

Comment:
Can this get more detail. As a user I would not understand this. Here are a few concrete suggestions:
* Call out the new fields
* Indicate which remote servers these fields can work with, e.g. `galaxy.ansible.com` and `cloud.redhat.com`.
* Relate the new fields to the ansible-galaxy CLI documentation here https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#configuring-the-ansible-galaxy-client